docs/content/get-started/build-workers.md

```diff
@@ -2,30 +2,52 @@
+<br/>
+<br/>
+3. **It has finished building when:** You see the following message in the Unity Editor's Console window: `Completed build for Cloud target`. 
 
-![SpatialOS menu in Unity]({{assetRoot}}assets/unity-spatialos-menu.png)
+>**TIP:** You may get a number of warnings displayed in the Unity Editor Console window. You can ignore the warnings at this stage; use the message icons on the right-hand side of the Console window to set it to display only info and error messages so you can see only the relevant messages.
 
-> **It has finished building when:** You see the following message in the Unity Editor's Console window: `Completed build for Cloud target`
+**If you encounter build errors:** 
+<br/>You might not have selected the build supports your game needs during your Unity setup.
 
-<%(Callout type="alert" message="If you encounter build errors, you might not have selected the build supports your game needs during your Unity setup. <br/><br/>
-* You need **Linux** build support. This is because server-workers in a cloud deployment run in a Linux environment. In the `Assets/Fps/Config/BuildConfiguration`, do not change the `UnityGameLogic Cloud Environment` from Linux.<br/> <br/>
+* You need **Linux** build support. This is because server-workers in a cloud deployment always run in a Linux environment. In the `Assets/Fps/Config/BuildConfiguration`, do not change the `UnityGameLogic Cloud Environment` from Linux.
 * You need **Mac** build support if you are developing on a Windows PC and want to share your game with Mac users.<br/>
 * You need **Windows** build support if you are developing on a Mac and want to share your game with Windows PC users. <br/>
-* Unity gives you build support for your development machine (Windows or Mac) by default.")%>
+* Unity gives you build support for your development machine (Windows or Mac) by default.
 
-After the build has successfully finished, the `gdk-for-unity-fps-starter-project/build/assembly` folder should contain:
+
+**After the build has successfully finished:** 
+<br/>Your `gdk-for-unity-fps-starter-project/build/assembly` folder should contain:
 ```text
   worker
       ├── SimulatedPlayerCoordinator@Linux.zip
       ├── UnityClient@Mac.zip
       ├── UnityClient@Windows.zip
       ├── UnityGameLogic@Linux.zip
 ```
+<br>
+
+>**TIP: Speed up development iteration with worker quick-run** 
+<br/> When you are developing with the GDK, you don't need to build out workers all the time, you can use quick-run to run multiple workers in your Unity Editor using Ctrl+L (Windows) or Cmd+L (Mac).
+<br/>
+<br/>When you are developing with the GDK, you use a local deployment rather than a cloud deployment. In a local deployment, you can either build your workers to run locally or use quick-run. With quick-run you can run multiple workers in your Unity Editor, so you don't have to keep building out workers during development iteration. 
```

Comment:
```suggestion
<br/>This tutorial takes you through the steps to set up a cloud deployment. When you are developing with the GDK, you use a local deployment rather than a cloud deployment. In a local deployment, you can either build your workers to run locally or use quick-run. With quick-run you can run multiple workers in your Unity Editor, so you don't have to keep building out workers during development iteration. 
```